_inc/client/traffic/verification-services.jsx

```diff
@@ -5,6 +5,8 @@ import React from 'react';
 import { translate as __ } from 'i18n-calypso';
 import TextInput from 'components/text-input';
 import ExternalLink from 'components/external-link';
+import Text from 'react-native';
```

Comment:
Can you tell us why we need `Text` here? It looks like it's not used anywhere in the scope.